neat_core.c

```diff
@@ -1078,7 +1078,9 @@ neat_change_timeout(neat_ctx *mgr, neat_flow *flow, int seconds)
         }
 
         return NEAT_ERROR_OK;
-    } else if (flow->sockProtocol == IPPROTO_SCTP) {
+    }
+#if defined(IPPROTO_SCTP)
```

Comment:
That was the intention.

> Am 17.06.2016 um 12:01 schrieb Michael Tüxen <notifications@github.com>:
> 
> In neat_core.c <https://github.com/NEAT-project/neat/pull/93#discussion_r67487325>:
> 
> > @@ -1078,7 +1078,9 @@ neat_change_timeout(neat_ctx *mgr, neat_flow *flow, int seconds)
> >          }
> >  
> >          return NEAT_ERROR_OK;
> > -    } else if (flow->sockProtocol == IPPROTO_SCTP) {
> > +    }
> > +#if defined(IPPROTO_SCTP)
> Is it possible to compile this code now even if IPPROTO_SCTP is not defined?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/NEAT-project/neat/pull/93/files/d1522cf0ed018d8427dd0c08f893f7f41e3e8f8c#r67487325>, or mute the thread <https://github.com/notifications/unsubscribe/AADsEhROJgnZIP_7Bi-UWrhYGgSEAo0qks5qMnCHgaJpZM4I3iJq>.
> 

*****************************************

Dr. Irene Rüngeler
Fachbereich Elektrotechnik und Informatik
Labor für Netzwerkprogrammierung, Datenbanklabor

Fachhochschule Münster
- University of Applied Sciences - 
Bismarckstr. 11, Raum 101
48565 Steinfurt

Tel: +49 2551 962-561 
Fax: +49 2551 962-563 
EMail: I.Ruengeler@fh-muenster.de